oaknational/Oak-Web-Application
# Feat/pupil 1040/change share behaviour

## Description

Music year: 2019

### Job Story

**When** a user clicks share,
**I want to instantly create and link and copy url for the share page; and in the background store the lesson attempt data; and on the share page add a loading state to give the data a chance to store if the user uses the link before the data has been stored** 
**So that user experience is more intuitive**.

### Implementation details

- make share button instantly copy url to clipboard
- add loading state to share page
- style share button

### Acceptance criteria

- [x]  share button has no loading state
- [x]  share page has to attempts to load data
- [x]  share buttons are styled

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2829--oak-web-application.netlify.thenational.academy
2. Click on a lesson and complete a lesson to the review page
3. You should see a share button, when pressed should instantly inform user the the share url has been copied to clipboard
4. and all button styles are correct

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
